plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/xsd/com.google.cloud.tools.eclipse.appengine.libraries.xsd

```diff
@@ -202,6 +205,51 @@ unspecified, the javadoc will be downloaded from the artifact location if availa
       </xs:complexType>
    </xs:element>
 
+   <xs:element name="enablement">
+      <xs:annotation>
+         <xs:documentation>
+            An Eclipse core expressions enablement element.
+            See http://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fcore%2Fexpressions%2Fpackage-summary.html
+            This schema is very restricted and needs to be expanded.
```

Comment:
This text should define what is the default variable.  I assume it's an `org.eclipse.core.resources.IProject`?